### ISSUE TITLE
Rewrite Inspector accordion as HTML fieldsets for a11y

### ISSUE THREAD

**Issue by mapk** [WordPress/gutenberg#13843]
**Problem**
Many a11y comments have touched on the need for proper labels for a set of controls. In the context of the Inspector, this might require the need to repeat the accordion title within the accordion itself which isn't fun. [An example of this is in this comment](https://github.com/WordPress/gutenberg/issues/13363#issuecomment-461277840) for responsive block controls.

**Possible Solution**
Would it be helpful to reevaluate the accordion markup in the Inspector and build them using the native [`fieldset` markup](https://accessibility.blog.gov.uk/2016/07/22/using-the-fieldset-and-legend-elements/)? If this happened can they still be accordion-like, with open and close interactions?

The benefit with this solution is that the label of the accordion would inform screenreaders of the purpose to the controls within the accordion itself.

_If there are problems with this proposed solution, please feel free to close._

**Comment by afercia** [comment]
Quickly discussed during [latest accessibility bug-scrub on Slack](https://wordpress.slack.com/archives/C02RP4X03/p1551024966199500), not sure this would be a good approach. Fieldset/legend are not always appropriate, it really depends on a case by case basis. 

While they're valuable for groups of radio buttons or checkboxes, it's arguable they would add benefit for large groups of form controls like the ones proposed on #13363. The following is a proposal from #13363:
 
<img width="826" alt="52791958-2a731e00-3038-11e9-9b58-eb046495fb04" src="https://user-images.githubusercontent.com/1682452/53753610-f7db7900-3eb1-11e9-916b-9f41dc6ec124.png">

Not appropriate for so large groups of _different types_ of form controls. Also, developers may want to add any kind of content within the panel accordions and enfording fieldset/legend would be incorrect.

Basically, fieldset and legend are appropriate when the _legend_ is (or is equivalent) to a short question the form controls grouped in the fieldset answer to. See https://accessibility.blog.gov.uk/2016/07/22/using-the-fieldset-and-legend-elements/

Suggest close 🙂

**Comment by mapk** [comment]
Thanks @afercia. Closing this issue.
